packages/storybook-addon/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@module-federation/storybook-addon",
+  "version": "0.0.1",
+  "description": "Storybook addon to consume remote module federated apps/components",
+  "license": "MIT",
+  "repository": "https://github.com/module-federation/universe/tree/main/packages/storybook-addon",
+  "keywords": [
+    "module-federation",
+    "typescript",
+    "storybook",
+    "addon"
+  ],
+  "author": "Fiodorov Andrei <hello@fyodorovandrei.com> (https://github.com/fyodorovandrei)",
+  "devDependencies": {
+    "@storybook/core-common": "^6.5.16",
+    "@storybook/node-logger": "^6.5.16",
+    "webpack": "^5.75.0",
+    "webpack-virtual-modules": "^0.5.0",
+    "@module-federation/utilities": "*"
```

Comment:
I think it would be safe to pin a specific version. Lets pin it to the latest version available now.